.github/workflows/build-and-test-snap.yml

```diff
@@ -25,4 +25,14 @@ jobs:
         with:
           name: ${{ steps.build.outputs.snap }}
           path: ${{ steps.build.outputs.snap }}
+          
+  test:
+    needs: build
+    runs-on: ubuntu-latest
+    steps:
+      - name: Download and test snap
+        uses: MonicaisHer/edgex-snap-testing/test@IENG-881
```

Comment:
Here is testing this test suite: https://github.com/MonicaisHer/matter-snap-testing/tree/IENG-881/test/suites/chip-tool